.github/dependabot.yml

```diff
@@ -5,58 +5,7 @@ updates:
     schedule:
       interval: weekly
 
-  - package-ecosystem: gomod
-    directory: /api
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /client/pkg
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /client/v2
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /client/v3
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /etcdctl
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /etcdutl
-    schedule:
-      interval: weekly
-
   - package-ecosystem: gomod
     directory: /
     schedule:
       interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /pkg
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /server
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
-    directory: /tests
-    schedule:
-      interval: weekly
-
-  - package-ecosystem: gomod
```

Comment:
I think that this one should stay -> It's not linked from the main go.mod.

Reply:
Good catch!